### ISSUE TITLE
Build error regarding "Headers" name not found when using this sdk.

### ISSUE THREAD

**Issue by quldude** [twilio/twilio-node#923]
### Issue Summary
Getting an error when building a nodejs app using twilio sdk.

### Steps to Reproduce
1. Use "twilio": "^4.8.0"
2. run build command `tsc -p tsconfig.build.json`

### Exception/Log
```
node_modules/twilio/lib/base/BaseTwilio.d.ts:22:19 - error TS2304: Cannot find name 'Headers'.

22         headers?: Headers;
                     ~~~~~~~


Found 1 error in node_modules/twilio/lib/base/BaseTwilio.d.ts:22
```

### Technical details:
* twilio-node version: 4.8.0
* node version: v18.12.1

**Comment by quldude** [comment]
resolved this by adding `DOM` in tsconfig.

**Comment by ehaynes99** [comment]
This should not be closed. Node projects should not include the `DOM` types. There is no `window`, `document`, etc. so you're opening up a whole class of runtime errors that should be compile errors. The twilio client should mimic the type and export it with the client.

**Comment by childish-sambino** [comment]
👀  on https://github.com/twilio/twilio-node/pull/925 which should fix this ...

**Comment by childish-sambino** [comment]
Fixed by #925
